lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -302,6 +302,33 @@ static bool recursiveType(const StructType *ST, const Type *Ty) {
   return Run(Ty);
 }
 
+// Add decoration if needed
+void addFPBuiltinDecoration(SPIRVModule *BM, Instruction *Inst,
+                            SPIRVInstruction *I) {
+  const bool AllowFPMaxError =
+      BM->isAllowedToUseExtension(ExtensionID::SPV_INTEL_fp_max_error);
+  if (!AllowFPMaxError)
+    return;
```

Comment:
Does it seem better?
```suggestion
  if (!BM->isAllowedToUseExtension(ExtensionID::SPV_INTEL_fp_max_error))
    return;
```

Reply:
Yes sure, thanks will do!